spec/controllers/settings_controller_spec.rb

```diff
@@ -1,28 +1,23 @@
 require 'spec_helper'
 
-describe SessionsController do
+describe SettingsController, type: :controller do
```

Comment:
I noticed that the 'settings_controller_spec.rb' really contained a partial SessionsController test instead of a SettingsController :)